<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourcesDialog</class>
 <widget class="QDialog" name="SourcesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVAMP: Load</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QPushButton" name="referenceButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a FASTA file to load</string>
     </property>
     <property name="text">
      <string>Local file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="annoButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a GFF file to load</string>
     </property>
     <property name="text">
      <string>Local file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="varButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a VCF file to load</string>
     </property>
     <property name="text">
      <string>Local file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Give a local file or a URL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="referenceLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a FASTA file to load</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="annoLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a GFF file to load</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="varLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a VCF file to load</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="referenceLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a FASTA file to load</string>
     </property>
     <property name="text">
      <string>Reference (fasta)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="annoLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a GFF file to load</string>
     </property>
     <property name="text">
      <string>Annotation (gff)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="varLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select a VCF file to load</string>
     </property>
     <property name="text">
      <string>Variation (vcf)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" rowspan="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" rowspan="2">
    <widget class="QPushButton" name="okButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="filterLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Filter the input files by chromosome for speed</string>
     </property>
     <property name="text">
      <string>Chromosome Filter</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="filterLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Filter the input files by chromosome for speed</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QTableWidget" name="chromosomeData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="5" column="3" rowspan="2">
    <widget class="QPushButton" name="infoButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sample Information(.db)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2" colspan="4">
    <widget class="QLineEdit" name="dbLine">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="dbButton">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Local File</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>referenceButton</zorder>
  <zorder>annoButton</zorder>
  <zorder>varButton</zorder>
  <zorder>titleLabel</zorder>
  <zorder>referenceLine</zorder>
  <zorder>annoLine</zorder>
  <zorder>varLine</zorder>
  <zorder>referenceLabel</zorder>
  <zorder>annoLabel</zorder>
  <zorder>varLabel</zorder>
  <zorder>cancelButton</zorder>
  <zorder>okButton</zorder>
  <zorder>filterLabel</zorder>
  <zorder>filterLine</zorder>
  <zorder>chromosomeData</zorder>
  <zorder>infoButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>label</zorder>
  <zorder>dbLine</zorder>
  <zorder>dbButton</zorder>
 </widget>
 <tabstops>
  <tabstop>referenceLine</tabstop>
  <tabstop>referenceButton</tabstop>
  <tabstop>annoLine</tabstop>
  <tabstop>annoButton</tabstop>
  <tabstop>varLine</tabstop>
  <tabstop>varButton</tabstop>
  <tabstop>filterLine</tabstop>
  <tabstop>infoButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>chromosomeData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
